sys/kern/pci.c

```diff
@@ -50,6 +50,8 @@ static uint32_t pci_bar_size(device_t *pcid, int bar, uint32_t addr) {
 
 DEVCLASS_CREATE(pci);
 
+#define INVALID_BAR 0xff
```

Comment:
I know PCI_BAR_MAX is equal to 6, but maybe it would be better to define invalid bar to be (-1)? 0xff seems a bit random, since rid is an int.

Reply:
I chose `0xff` because `nbars` field of `pci_device_t` is an `uint8_t`.

Speaking of which, I'm unsure whether I should keep it. Technically it isn't really used by any of existing drivers, but it might be considered a useful information.